contracts/accountants/HealthCheckAccountant.sol

```diff
@@ -320,7 +366,7 @@ contract HealthCheckAccountant {
         uint16 customMaxFee,
         uint16 customMaxGain,
         uint16 customMaxLoss
-    ) external onlyFeeManager {
+    ) external virtual onlyFeeManager {
```

Comment:
small suggestion for natspec: 

on `updateDefaultConfig` instead of "used for all strategies", i would say: "... effective for all strategies that don't already have a custom config set"

Reply:
https://github.com/yearn/vault-periphery/pull/27/commits/b0aa4889148b14212c0914a3276e430c345b22ef